arrow-libs/core/arrow-meta/arrow-meta-test-models/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+import org.jetbrains.kotlin.gradle.dsl.ExplicitApiMode
+
+plugins {
+  id(libs.plugins.arrowGradleConfig.jvm.get().pluginId)
```

Comment:
Do we need this here?

Before it was just using, which bypasses the configuration from Arrow and just turns this into a vanilla JVM module.

```
id "org.jetbrains.kotlin.jvm"
id "org.jetbrains.kotlin.kapt"
```

Reply:
Thank you for this one 💯